app/controllers/better_together/notifications_controller.rb

```diff
@@ -11,11 +11,11 @@ def index
     end
 
     # TODO: Make a Stimulus controller to dispatch this action async when messages are viewed
-    # rubocop:todo Metrics/MethodLength
     def mark_as_read # rubocop:todo Metrics/AbcSize, Metrics/MethodLength
       if params[:id]
-        @notification = helpers.current_person.notifications.find(params[:id])
-        @notification.update(read_at: Time.current)
+        mark_notification_as_read(params[:id])
+      elsif params[:message_id]
```

Comment:
I'm thinking that it might be a good idea to rewrite this part to use `record` and `record_id` instead of a reference specifically to message_id. Looking at the implementation below, there would be no real difference between marking `message_notifications` as read versus marking `record_notifications`.

You may want to also adjust the method name to something that clarifies that the notification of a record  (or message) is being marked read, not the record (or message) itself.